### ISSUE TITLE
WARNING - Yolink International variants

### ISSUE THREAD

**Issue by LindsayReid** [home-assistant/home-assistant.github.io#28520]
### Feedback

I understand from Yolink that the Home Assistant integration does not currently work with the EU/UK variants of their products with the "E" suffix. They will only work with the "U" suffix for the USA variety.

### URL

https://www.home-assistant.io/integrations/yolink/

### Version

2023.8.1

**Comment by LindsayReid** [comment]
This is still in the hands of Yolink to sort out as it is outstanding. The issue is not publicised by YoLink and this is one of the few places that has flagged that the Non US versions of their devices do not work with Home Assistant.

**Comment by Bigboldbri** [comment]
It is very frustrating that the Yolink EC versions do not work with Home assistant. The EC version of the YS1603 hub runs firmware 0351 while the UC version is 0361.
Since ordering the YS1603-UC hub I have discovered it will not work with the YS7805-EC motion sensors , presumably I need the -UC motion sensor version to match the -UC Hub.....

**Comment by LindsayReid** [comment]
Sorry to say that you quite right!

 The UC and EC versions work on different frequencies. You can thus only use the UC hub with UC sensors/devices and the EC hub with EC sensors/devices. This is due to differing international frequency allocations. Why on earth Yolink can't just sort out the EC  firmware, I don't know. They said they were going to do it months ago.

**Comment by jimmyeao** [comment]
I've emailed Yolink today to ask them again when they plan to add support, I'll post any replies I get here

**Comment by jimmyeao** [comment]
Firmware update on my Hub to 0406 and is now working in HA

**Comment by davidrule1969** [comment]
> Firmware update on my Hub to 0406 and is now working in HA

Where did you get firmware 0406 from?  The Yolink app is telling me that my 0356 firmware is the latest available.  Perhaps I just have to be patient! :-)

**Comment by jimmyeao** [comment]
> > Firmware update on my Hub to 0406 and is now working in HA
> 
> Where did you get firmware 0406 from? The Yolink app is telling me that my 0356 firmware is the latest available. Perhaps I just have to be patient! :-)

it just appeared in my app, maybe try force quit / re log in the app? there was no notification, I just tapped where it said firmware then it asked me if I wanted to update

**Comment by LindsayReid** [comment]
> > > Firmware update on my Hub to 0406 and is now working in HA

Jimmy, can you confirm you are in UK/EU and the Yolink devices are the EU/EC versions and Not the US/UC ones? Many thanks.

**Comment by jimmyeao** [comment]
> > > > Firmware update on my Hub to 0406 and is now working in HA
> 
> Jimmy, can you confirm you are in UK/EU and the Yolink devices are the EU/EC versions and Not the US/UC ones? Many thanks.

Yes, I am in UK and my Yolink devices are EU versions (Example, temp/humidity is model YS8003-EC)
![image](https://github.com/home-assistant/home-assistant.io/assets/5197831/8c412aaf-f013-41e5-aff2-b0d594ec6b70)

![image](https://github.com/home-assistant/home-assistant.io/assets/5197831/1b76b4eb-9ad5-44c0-ae59-009725535368)

![image](https://github.com/home-assistant/home-assistant.io/assets/5197831/b584f235-0deb-40a6-acbe-89d793edbef1)

**Comment by LindsayReid** [comment]
Many thanks - Good to know they've sorted it at last,
